tests/unit/s2n_self_talk_client_hello_cb_test.c

```diff
@@ -39,12 +39,14 @@ int mock_client(int writefd, int readfd, int expect_failure)
     s2n_blocked_status blocked;
     int result = 0;
     int rc = 0;
+    const char *protocols[] = { "http/2", "http/1.1" };
```

Comment:
I believe it is just h2 for http/2.
But this is just a test, so meh.

But I guess someone looking to implement http/2 May see this and get confused....

Reply:
Yup it has to be "h2". I will update it with other changes (if any).